uvdb/VITABUILD

```diff
@@ -0,0 +1,19 @@
+pkgname=uvdb
+pkgver=9999
+pkgrel=1
+url="https://github.com/sleirsgoevy/vita-uvdb"
+source=("git+https://github.com/sleirsgoevy/vita-uvdb.git")
```

Comment:
Same here as above

Reply:
Should I actually **decrement** the version here?

Reply:
tag vita-uvdb (for example 1.0, or 0.1, or whatever), and use same version for pkgver (`pkgver=1.0`)
point source to said git tag

Reply:
Done.

The question about version downgrade was about the kubridge package, which is already in the repo.